### PR TITLE
Latest images

### PR DESCRIPTION
**Pull request by angelk**
Show latest images on index page

### CODE REVIEW

File: translations/messages.bg.php [hunk 1]
```diff
@@ -19,6 +19,8 @@
     'Latest tracks' => 'Най-нови следи',
     'Latest tracks by bicycle' => 'Най-нови следи с колело',
     'Latest tracks by foot' => 'Най-нови пешеходни следи',
+    'Latest pictures' => 'Най-нови снимки',
```
Comment (StefanMavrodiev) [inline translations/messages.bg.php]: Here _pictures_ is _снимки_

File: translations/messages.bg.php [hunk 2]
```diff
@@ -34,6 +36,8 @@
     'Add new version' => 'Добави нова версия',
     'Edit' => 'Промени',
     'Short link' => 'Кратък линк',
+    'Picture from' => 'Картинка от',
```
Comment (StefanMavrodiev) [inline translations/messages.bg.php]: Here _picture_ is _картинка_

File: templates/home/home.html.twig [hunk 3]
```diff
@@ -45,6 +45,25 @@
                 </ul>
             </div>
         </div>
+
+        {% if latestImages is defined and latestImages is not empty %}
+            {{ 'Latest pictures'|trans }}:
+            <div class="container-fluid track_images">
```
Comment (StefanMavrodiev) [inline templates/home/home.html.twig]: If you put images in container only, they are not centred and there is a black spot on the right.

You could use flex box for this. For example wrap the images inside `<div class="d-flex justify-content-between">`

Comment (angelk) [reply templates/home/home.html.twig]: container-fluid is removed.
If I add d-flex then there is no space between the images.

File: translations/messages.bg.php (hunk 2 above)
Comment (angelk) [reply translations/messages.bg.php]: fixed

File: templates/home/home.html.twig (hunk 3 above)
Comment (StefanMavrodiev) [reply templates/home/home.html.twig]: That's why you should add _justyfy-content-between_ or some margin.

Anyway, the images are still ugly, at least on my screen:
Screenshot from 2019-09-16 23-42-31

Comment (angelk) [reply templates/home/home.html.twig]: fixed

**Comment by StefanMavrodiev** [comment]
Have you tried small viewport. Seems like images are missing?

**Comment by angelk** [comment]
Yes, works on pixel emulation
gps test_en(Pixel 2 XL)

**Pull request merged**